projects/storefrontapp/src/environments/environment.ts

```diff
@@ -12,6 +12,7 @@ export const environment: Environment = {
     'https://spartacus-devci767.eastus.cloudapp.azure.com:9002',
   // 'https://spartacus-dev0.eastus.cloudapp.azure.com:9002',
   occApiPrefix: build.process.env.SPARTACUS_API_PREFIX ?? '/occ/v2/',
+  productconfig: true,
```

Comment:
It should also use env variables here

Reply:
@Platonn Do you agree? Didn't we say that as additive feature (like storefinder) product config should always be active

Reply:
Agreed on this approach:
Introduce env variable but default to true
Also rename `productconfig` to `productConfig`